mender_test.go

```diff
@@ -24,7 +25,11 @@ func Test_updateState_readBootEnvError_returnsError(t *testing.T) {
 	fakeEnv := uBootEnv{&runner}
 	mender := NewMender(&fakeEnv)
 
-	if err := mender.updateState(); err == nil {
+	// pretend we're boostrapped
+	mender.state = MenderStateBootstrapped
+
+	if state := mender.TransitionState(); state != MenderStateError {
+		t.Logf("got state: %v", state)
 		t.FailNow()
```

Comment:
Or that. Before I knew about that one I started using `mendertesting.AssertTrue`, but testify is probably more complete.